parser.h

```diff
@@ -6,11 +6,43 @@
 #include <any>
 #include <vector>
 
+/**
+ * \class Parser
+ * 
+ * \brief Parser class
+ * 
+ * \author poloczmate
+ *
+ * \date 2020.10.28. 21:00
+ *
+ * \note The Parser can read data from JSON files
+ * 
+ */
+
 class Parser{
-private:
-    static std::string FindData(const std::string& StringToParse, const std::string& StringToFind);
 public:
+/**
+	 * \brief This method can search a std::string and get the useful informations.
+	 * \return return with a std::string
+	 * \throw throw a "runtime_error" if something wrong
+	 */
+    static std::string FindData(const std::string& StringToParse, const std::string& StringToFind);
```

Comment:
Itt és minden más esetben ahol ez szerepel, ki kellene fejteni, hogy mit értetek a "the useful information" alatt.

Reply:
Köszi, javítva!